site/static/diagrams/gameserver-states.dot

```diff
@@ -38,6 +44,7 @@ digraph {
     RequestReady -> Unhealthy [ color="purple" ]
```

Comment:
The Scheduled -> Unhealthy arrow should be purple, right? It's on line 43 in the new file.

Reply:
Yeah, I wondered about that too. Fixed.